sdk/core/azure-core/azure/core/utils/_utils.py

```diff
@@ -88,6 +88,8 @@ def _convert_to_isoformat(date_time):
 
 def case_insensitive_dict(*args: Any, **kwargs: Any) -> MutableMapping[str, Any]:
     """Return a case-insensitive mutable mapping from an inputted mapping structure.
+    :param args: The input mapping structure.
+    :type args: Mapping[str, Any] or Iterable[Tuple[str, Any]
```

Comment:
for this typing of args will come in the mypy update ?

Reply:
is the second supposed to be for the kwargs ?

Reply:
It is for args. Same as the one in azure.core.utils.CaseInsensitiveDict